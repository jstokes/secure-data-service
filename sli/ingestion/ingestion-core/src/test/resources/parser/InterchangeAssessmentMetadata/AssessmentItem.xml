<?xml version="1.0" encoding="UTF-8"?>
<InterchangeAssessmentMetadata xmlns:sli="http://slc-sli/ed-org/0.1" xmlns="http://ed-fi.org/0100"
  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
  <AssessmentItem id="ID001">
    <AssessmentReference id="ID003" ref="ID001">
      <AssessmentIdentity>
        <AssessmentTitle>vj8zbHeMTeYFobCU0TF21q-Cy8n</AssessmentTitle>
        <AcademicSubject>Hospitality and Tourism</AcademicSubject>
        <GradeLevelAssessed>Tenth grade</GradeLevelAssessed>
        <Version>-1179495731</Version>
      </AssessmentIdentity>
    </AssessmentReference>
    <IdentificationCode>M3Pfl1._SPeQLaKiJiEqcQIZTPBpGJstVLq03QwcjJdiGwBWe_</IdentificationCode>
    <ItemCategory>Matching</ItemCategory>
    <MaxRawScore>-1973159586</MaxRawScore>
    <CorrectResponse>X3uT5DZ</CorrectResponse>
    <LearningStandardReference>
      <LearningStandardIdentity>
        <IdentificationCode>gpF8VKfk7Gf4.vEuUdv9n.I3iGiMucJ7</IdentificationCode>
      </LearningStandardIdentity>
    </LearningStandardReference>
    <Nomenclature>JdVo8tkNO</Nomenclature>
  </AssessmentItem>
</InterchangeAssessmentMetadata>
