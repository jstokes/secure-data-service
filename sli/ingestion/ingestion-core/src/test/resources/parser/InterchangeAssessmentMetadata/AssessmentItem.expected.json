{
  "a_id":"ID001",
  "AssessmentReference":{
    "a_id":"ID003",
    "a_ref":"ID001",
    "AssessmentIdentity":{
      "AssessmentTitle":{"_value":"vj8zbHeMTeYFobCU0TF21q-Cy8n"},
      "AcademicSubject":{"_value":"Hospitality and Tourism"},
      "GradeLevelAssessed":{"_value":"Tenth grade"},
      "Version":{"_value":-1179495731}
    }
  },
  "IdentificationCode":{"_value":"M3Pfl1._SPeQLaKiJiEqcQIZTPBpGJstVLq03QwcjJdiGwBWe_"},
  "ItemCategory":{"_value":"Matching"},
  "MaxRawScore":{"_value":-1973159586},
  "CorrectResponse":{"_value":"X3uT5DZ"},
  "LearningStandardReference":[{
    "LearningStandardIdentity":{
      "IdentificationCode":{"_value":"gpF8VKfk7Gf4.vEuUdv9n.I3iGiMucJ7"}
    }
  }],
  "Nomenclature":{"_value":"JdVo8tkNO"}
}
